src/components/AddNewUser.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<!-- Что за фокус с native, как это работает? -->
+
+	<el-form
+		@submit.native.prevent="onSubmit"
+		ref="form"
+		:model="form"
+		label-width="120px"
+	>
+		<el-form-item label="Fullname">
+			<el-input :placeholder="fullNamePH" v-model="form.fullName"></el-input>
```

Comment:
`<el-input :placeholder="fullNamePH" v-model="form.fullName"></el-input>` try to use self closing tag `<el-input :placeholder="fullNamePH" v-model="form.fullName" />`